app.ts

```diff
@@ -336,9 +336,13 @@ const httpRoot = urljoin(ceProps('httpRoot', '/'), '/');
 const staticUrl = ceProps<string | undefined>('staticUrl');
 const staticRoot = urljoin(staticUrl || urljoin(httpRoot, 'static'), '/');
 
-function staticHeaders(res: express.Response) {
+/**
+ * Sets appropriate headers for static resources. This is based on the staticMaxAgeSecs property from the
+ * compiler-explorer.properties file.
+ */
+export function addStaticHeaders(res: express.Response) {
```

Comment:
I saw no reason as to why this shouldn't be exported, instead of passed around like crazy

Reply:
Agreed: thanks. I think this is another one of my ideas for making it either testable (which it isn't) or some kind of flexibility we don't ultimately want or need.